nix/overlay.nix

```diff
@@ -3,4 +3,5 @@ self: super: {
   control-plane = super.callPackage ./pkgs/control-plane { };
   utils = super.callPackage ./pkgs/utils { };
   openapi-generator = super.callPackage ./pkgs/openapi-generator { };
+  control-plane-cov = (super.callPackage ./pkgs/control-plane { }).cov;
```

Comment:
can be deleted as you're not using this AFAICT

Reply:
Done.